docs/snippets/common/visual-test-addon-install.npx.js.mdx

```diff
@@ -0,0 +1,3 @@
+```shell
+npx storybook@latest add @chromaui/addon-visual-tests
```

Comment:
@kylegach, heads up: when addressing these snippets, they shouldn't be suffixed with `next` as this is a satellite repository and doesn't contain a `next` branch that allows users to install pre-release versions similar to other addons.

Reply:
I double-checked the logic; only `@storybook/...` packages are affected.

Reply:
Appreciate you taking the time to check this, glad that we don't need to do anything extra.